handle_hosts.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/zazab/hierr"
+)
+
+func handleHosts(
+	zabbix *Zabbix,
+	config *Config,
+	args map[string]interface{},
+) error {
+	var (
+		hostnames, _ = parseSearchQuery(args["<pattern>"].([]string))
+		//addHost, _    = args["--add"].(string)
+		removeHost, _ = args["--remove"].(string)
+		confirmation  = !args["--noconfirm"].(bool)
+
+		hostids = []string{}
+		table   = tabwriter.NewWriter(os.Stdout, 1, 4, 2, ' ', 0)
+	)
+
+	var (
+		err               error
+		hosts, foundHosts []Host
+		hostid            string
+		messages          string
+	)
+
+	uniqKeys := make(map[string]bool)
+
+	switch {
+	case removeHost != "":
+
+		foundHosts, err = searchHosts(zabbix, removeHost)
+		if err != nil {
+			return hierr.Errorf(
+				err,
+				"can't obtain zabbix hosts",
+			)
+		}
+		for len(foundHosts) == 0 {
+			return nil
+		}
+
+		for len(foundHosts) > 1 {
+			return fmt.Errorf(
+				"found more then one uniq host for hostname: %s",
```

Comment:
You should use `karma-go` to describe what hosts or its quantity was found by the search pattern.